messenger.rst

```diff
@@ -1219,7 +1219,7 @@ this globally (or for each transport) to a service that implements
                 ],
                 'transports' => [
                     'async_priority_normal' => [
-                        'dsn' => // ...
+                        'dsn' => ...,
```

Comment:
Valid code should be `'dsn' => '...',` right?

Reply:
Yes, You are correct. 

But Im actually a bit sneaky here. (Still not sure if it is "good sneaky" or "bad sneaky"). I convert all `...` to `null`, (see [code](https://github.com/symfony-docs-tools/code-block-checker/blob/886c6ad47df563c62944d8dd5e5c1043ac630293/src/Service/CodeValidator/PhpValidator.php#L26)) that makes it easy to write examples and the code is still valid.

Reply:
It makes a lot of sense. Better not be too strict and allow using `...`, which is very common in docs. Thanks.